dcrMetric.py

```diff
@@ -0,0 +1,301 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import scipy.stats
+
+
+class DcrMetric:
+    def __init__(self, max_airmass=1.8, width=0.4):
+        self.max_airmass = max_airmass
+        self.range = [1 - max_airmass, max_airmass - 1]
+        self.width = width
+        self.kde = None
+        self.density = None
+        self.norm = None
+        self.metric = None
+        self.table = pd.DataFrame(columns=['airmass', 'hour_angle'], dtype='float')  # visit will be the index
+        self.metric_reference = self.calculateReference()
+                
+    def parameterizeVisit(self, airmass, hour_angle):
+        """Convert airmass and hour angle to visit measure.
+        
+        Parameters
+        ----------
+        airmass: `float`
+            The observed airmass of the observation.
+        hour_angle: `float`
+            The observed hour angle of the observation.
+        
+        Returns
+        -------
+        visit_measure: `float`
+            A one dimensional parameterization of airmass and hour angle.
+        """
+        visit_measure = (airmass - 1.)*np.sign(hour_angle)
+        return visit_measure
+    
+    def parameterizeTable(self, visits=None):
```

Comment:
Here and throughout, you switch between calling the main pandas dataframe you're using a "table" and a "database." Maybe just call it a dataframe? At the very least, please be consistent.